modules/dashboard.py

```diff
@@ -32,6 +33,8 @@ def __init__(self, hardware: HardwareModule, module_list: ModuleList):
         putCommandOnDashboard("Claw", Drop.atLevel3(hardware.claw))
         putCommandOnDashboard("Claw", Drop.atLevel4(hardware.claw))
 
+        putCommandOnDashboard("Auto Drop", AutoDrop(hardware.claw, hardware.elevator))
```

Comment:
On veut le nom du groupe en premier, c'est-à-dire le nom du subsystem (ici, `Claw`).
```suggestion
        putCommandOnDashboard("Claw", AutoDrop(hardware.claw, hardware.elevator))
```